Admin.js

```diff
@@ -34,78 +35,104 @@ var Admin = {
         Admin.setup_sticky_elements(subject);
         Admin.setup_readmore_elements(subject);
 
-//        Admin.setup_list_modal(subject);
+        //        Admin.setup_list_modal(subject);
+
     },
-    setup_list_modal: function(modal) {
-        Admin.log('[core|setup_list_modal] configure modal on', modal);
-        // this will force relation modal to open list of entity in a wider modal
-        // to improve readability
-        jQuery('div.modal-dialog', modal).css({
-            width:  '90%', //choose your width
-            height: '85%',
-            padding: 0
+    "setup_list_modal" (modal) {
+
+        Admin.log("[core|setup_list_modal] configure modal on", modal);
+
+        /*
+         * This will force relation modal to open list of entity in a wider modal
+         * to improve readability
+         */
+        jQuery("div.modal-dialog", modal).css({
+            "width": "90%", // Choose your width
+            "height": "85%",
```

Comment:
- eslint: Expected object keys to be in ascending order. 'height' should be before 'width'.